topics/lru-cache.md

```diff
@@ -81,6 +85,13 @@ So we continuously cross the boundaries of the memory limit, by going over it, a
 
 If a command results in a lot of memory being used (like a big set intersection stored into a new key) for some time, the memory limit can be surpassed by a noticeable amount.
 
+## Monitor eviction
+
+To monitor the point when Valkey starts to evict data, use the `INFO MEMORY` output. Compare the current memory usage displayed in the `used_memory` output with the `maxmemory` value. 
```

Comment:
is this correct tho? `used_memory` appears to include replication buffers and the like, whereas `used_memory_dataset` seems to account for the "output buffers needed to feed the replicas are subtracted from the used memory count" part.